app/src/androidTest/java/com/hedvig/app/feature/loggedin/NavBarNoReferralTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.hedvig.app.feature.loggedin
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.agoda.kakao.screen.Screen
+import com.hedvig.android.owldroid.graphql.LoggedInQuery
+import com.hedvig.app.feature.loggedin.ui.LoggedInActivity
+import com.hedvig.app.testdata.feature.referrals.LOGGED_IN_DATA_WITH_REFERRALS_FEATURE_DISABLED
+import com.hedvig.app.util.ApolloCacheClearRule
+import com.hedvig.app.util.ApolloMockServerRule
+import com.hedvig.app.util.apolloResponse
+import com.hedvig.app.util.hasNumberOfMenuItems
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class NavBarNoReferralTest {
```

Comment:
Should we express this as a positive instead of a negative, i.e `NavBarKeyGearEnabledTest`?

Reply:
True that sounds better